services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+from db.models import MovieSession, Movie, CinemaHall
+from django.db.models import QuerySet
+from typing import Optional
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)

```

Comment:
No need to do get just use _id
```suggestion
        movie_id=movie_id,
        cinema_hall_id=cinema_hall_id

```